.github/workflows/ci.yaml

```diff
@@ -21,8 +26,10 @@ jobs:
 
       - name: Upgrade linux deps
         run: |
-          sudo apt-get update
-          sudo apt-get upgrade --yes
+          sudo apt update
+          # https://askubuntu.com/questions/1276111/error-upgrading-grub-efi-amd64-signed-special-device-old-ssd-does-not-exist
+          sudo apt-mark hold grub-efi-amd64-signed grub-efi-amd64-bin
+          sudo DEBIAN_FRONTEND=noninteractive apt upgrade --yes
```

Comment:
I actually wonder if it wouldn't be enough to only apply security updates instead of everything?